Source/JavaScriptCore/b3/air/testair.cpp

```diff
@@ -96,7 +96,7 @@ std::unique_ptr<Compilation> compile(B3::Procedure& proc)
     generate(proc.code(), jit);
     LinkBuffer linkBuffer(jit, nullptr);
 
-    return makeUnique<Compilation>(
+    return makeUniqueWithoutFastMallocCheck<Compilation>(
```

Comment:
I think you missed the `WTF_MAKE_SEQUESTERED_ARENA_ALLOCATED` for `class Compilation`.

Reply:
Ah, that's part of a different patch (which does not yet work) -- so this line isn't necessary for now. Will remove